x-pack/plugins/observability/public/pages/slo_edit/slo_edit.test.tsx

```diff
@@ -108,9 +109,13 @@ const mockKibana = () => {
         get: () => {},
       },
       triggersActionsUi: {
-        getAddRuleFlyout: jest
-          .fn()
-          .mockReturnValue(<div data-test-subj="add-rule-flyout">Add Rule Flyout</div>),
+        getAddRuleFlyout: jest.fn().mockReturnValue(
+          <div data-test-subj="add-rule-flyout">
+            {i18n.translate('xpack.observability.mockKibana.div.addRuleFlyoutLabel', {
+              defaultMessage: 'Add Rule Flyout',
+            })}
+          </div>
+        ),
```

Comment:
This change came from this Kibana machine auto-commit (1337ca26cd84642012a720422e768710643b0ae3) - notsure why this ended up in my PR, since I didn't touch this file? 🤷